true/false variables compile error

On platforms which have stdbool.h and the true/false as macros, things like below cause issues

> p11_attr.c: In function 'pkcs11_addattr_bool':
> p11_attr.c:126:18: error: expected identifier or '(' before numeric constant
>   static CK_BBOOL true = CK_TRUE;
>                   ^~~~
> p11_attr.c:127:18: error: expected identifier or '(' before numeric constant
>   static CK_BBOOL false = CK_FALSE;
>                   ^~~~~
> p11_attr.c:128:37: error: lvalue required as unary '&' operand
>   pkcs11_addattr(tmpl, type, value ? &true : &false, sizeof(CK_BBOOL));
>                                      ^
> p11_attr.c:128:45: error: lvalue required as unary '&' operand
>   pkcs11_addattr(tmpl, type, value ? &true : &false, sizeof(CK_BBOOL));

Maybe convert these to _false and _true or some other name to avoid name clashes?

--- a/src/p11_attr.c
+++ b/src/p11_attr.c
@@ -123,9 +123,9 @@ unsigned int pkcs11_addattr(PKCS11_TEMPLATE *tmpl, int type, void *data, size_t 
 
 void pkcs11_addattr_bool(PKCS11_TEMPLATE *tmpl, int type, int value)
 {
-	static CK_BBOOL true = CK_TRUE;
-	static CK_BBOOL false = CK_FALSE;
-	pkcs11_addattr(tmpl, type, value ? &true : &false, sizeof(CK_BBOOL));
+	static CK_BBOOL _true = CK_TRUE;
+	static CK_BBOOL _false = CK_FALSE;
+	pkcs11_addattr(tmpl, type, value ? &_true : &_false, sizeof(CK_BBOOL));
 }
 
 void pkcs11_addattr_s(PKCS11_TEMPLATE *tmpl, int type, const char *s)

--- a/src/p11_ec.c
+++ b/src/p11_ec.c
@@ -590,22 +590,22 @@ static int pkcs11_ecdh_derive(unsigned char **out, size_t *outlen,
 	CK_MECHANISM mechanism;
 	int rv;
 
-	CK_BBOOL true = TRUE;
-	CK_BBOOL false = FALSE;
+	CK_BBOOL _true = TRUE;
+	CK_BBOOL _false = FALSE;
 	CK_OBJECT_HANDLE newkey = CK_INVALID_HANDLE;
 	CK_OBJECT_CLASS newkey_class= CKO_SECRET_KEY;
 	CK_KEY_TYPE newkey_type = CKK_GENERIC_SECRET;
 	CK_ULONG newkey_len = key_len;
 	CK_OBJECT_HANDLE *tmpnewkey = (CK_OBJECT_HANDLE *)outnewkey;
 	CK_ATTRIBUTE newkey_template[] = {
-		{CKA_TOKEN, &false, sizeof(false)}, /* session only object */
+		{CKA_TOKEN, &_false, sizeof(_false)}, /* session only object */
 		{CKA_CLASS, &newkey_class, sizeof(newkey_class)},
 		{CKA_KEY_TYPE, &newkey_type, sizeof(newkey_type)},
 		{CKA_VALUE_LEN, &newkey_len, sizeof(newkey_len)},
-		{CKA_SENSITIVE, &false, sizeof(false) },
-		{CKA_EXTRACTABLE, &true, sizeof(true) },
-		{CKA_ENCRYPT, &true, sizeof(true)},
-		{CKA_DECRYPT, &true, sizeof(true)}
+		{CKA_SENSITIVE, &_false, sizeof(_false) },
+		{CKA_EXTRACTABLE, &_true, sizeof(_true) },
+		{CKA_ENCRYPT, &_true, sizeof(_true)},
+		{CKA_DECRYPT, &_true, sizeof(_true)}
 	};
 
 	memset(&mechanism, 0, sizeof(mechanism));

